### PR TITLE
Update to ALPHA7

### PR DESCRIPTION
**Pull request by Aericio**
#### **What does the PR do?**

Update to ALPHA7

#### **Why should this PR be merged?**

Updates plugin to work in ALPHA7

#### **Are the changes in this PR tested**?

- [x] Yes
- [ ] No

#### **Extra Information**

there might be more than just that, so i am creating a pr (unnecessary help salmon doesn't need, prob.)

### CODE REVIEW

File: src/SalmonDE/StatsPE/EventListener.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ public function onJoin(\pocketmine\event\player\PlayerJoinEvent $event){
         }
 
         if(Base::getInstance()->getDataProvider()->entryExists('ClientID')){
-            Base::getInstance()->getDataProvider()->saveData($event->getPlayer()->getName(), Base::getInstance()->getDataProvider()->getEntry('ClientID'), (string) $event->getPlayer()->getClientId());
+            Base::getInstance()->getDataProvider()->saveData($event->getPlayer()->getName(), Base::getInstance()->getDataProvider()->getEntry('ClientID'), (string) $event->getPlayer()->getClientId()); // getClientID() is deprecated
```
Comment (Aericio) [inline src/SalmonDE/StatsPE/EventListener.php]: as phpstorm says*

Comment (SalmonDE) [reply src/SalmonDE/StatsPE/EventListener.php]: Why is the comment needed there? No reason to have it there and this branch should only have commits related to the api change

**Review by SalmonDE — CHANGES_REQUESTED** [review]
Works, but the added comment in this pr should be removed.
Other than that, ready to merge.

**Comment by Aericio** [comment]
understood.

**Pull request merged**